default/data/ui/views/global-dashboard.xml

```diff
@@ -10,19 +10,16 @@
       </default>
     </input>
   </fieldset>
-  <!--Base search part start-->
```

Comment:
Could you merge this commit with the previous one ?

Reply:
I have a little trouble with merge this to commit in one